docs/src/components/copy-button.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
```

Comment:
don't import * from React, import the specific things you need from it.
```suggestion
import { useEffect, useState, type HTMLAttributes } from "react";
```

Reply:
I don't think it really matters here